registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java

```diff
@@ -1250,6 +1250,20 @@ public enum PlatformErrorMessages {
 	RPR_ERROR_IN_ABIS_HANDLER_IDENTIFY_REQUEST(PlatformConstants.RPR_ABIS_HANDLER + "004",
 			"Internal Error occured in Abis Handler identify request"),
 
+	RPR_ABIS_HANDLER_STAGE_FAILED(PlatformConstants.RPR_ABIS_HANDLER + "005", "ABIS Handler Stage failed"),
+
+	RPR_BIOMETRIC_RECORD_VALIDATION_FAILED(PlatformConstants.RPR_ABIS_HANDLER + "006",
+			"Biometric record validation error"),
+
+	RPR_DATASHARE_MODALITIES_EMPTY(PlatformConstants.RPR_ABIS_HANDLER + "007",
+			"Data Share Policy Modalities were Empty"),
+
+	RPR_NO_BIOMETRICS_FOUND_WITH_DATASHARE(PlatformConstants.RPR_ABIS_HANDLER + "008",
+			"No Biometrics Found with Data Share Policy"),
+
+	RPR_NO_BIOMETRIC_MATCH_WTIH_DATASAHRE(PlatformConstants.RPR_ABIS_HANDLER + "010",

```

Comment:
why 009 is missed ?

Reply:
Previously I added one more error message when I used Biometric config segment map ,removed that.Updated with latest number.